.env.example

```diff
@@ -1,22 +1,38 @@
 AWS_ACCOUNT=123456
 AWS_REGION=eu-west-2
 
-# API Specific
+# API
+
+## Application variables
 ALLOWED_EMAIL_DOMAINS=example1.com,example2.com
 DATA_BUCKET=the-bucket
 RESOURCE_PREFIX=rapid
 DOMAIN_NAME=example.com
 COGNITO_USER_POOL_ID=11111111
 LAYERS=raw,layer
 CUSTOM_USER_NAME_REGEX="regex_expression"
-# SDK Specific
+
+## Build variables
+ECS_SERVICE=rapid-ecs-service
```

Comment:
Just checking why we need these variables?

Reply:
We need them for a couple of the release commands, they do things like force redeploy the ECS service if you're releasing a new service and wait for the ECS service to be running